psci/PSCi.hs

```diff
@@ -124,17 +124,36 @@ expandTilde p = return p
 --
 helpMessage :: String
 helpMessage = "The following commands are available:\n\n    " ++
-  intercalate "\n    " (map line D.help)
+  intercalate "\n    " (map line D.help) ++
+  "\n\n" ++ extraHelp
   where
-    line :: (Directive, String, String) -> String
-    line (dir, arg, desc) = intercalate " "
-          [ cmd
-          , replicate (11 - length cmd) ' '
-          , arg
-          , replicate (11 - length arg) ' '
-          , desc
-          ]
-      where cmd = ":" ++ D.stringFor dir
+  line :: (Directive, String, String) -> String
+  line (dir, arg, desc) =
+    let cmd = ':' : D.stringFor dir
+    in intercalate " "
+        [ cmd
+        , replicate (11 - length cmd) ' '
+        , arg
+        , replicate (11 - length arg) ' '
+        , desc
+        ]
+
+  extraHelp = unlines
```

Comment:
I guess my preference would be to stick the extra content on the wiki with a link in `psci` honestly. I like that approach because it means we don't have to do a release to update textual content in the compiler, like we do for error messages now.